torch/distributed/algorithms/ddp_comm_hooks/default_hooks.py

```diff
@@ -31,17 +31,19 @@ def then_callback(fut):
     return fut.then(then_callback)
 
 
-def fp16_compress_hook(process_group: object, bucket: dist._GradBucket):
+def fp16_compress_hook(
+    process_group: object, bucket: dist._GradBucket
+) -> torch.futures.Future:
     """
-        This DDP communication hook implements a simple gradient compression
-        approach that converts ``GradBucket`` tensors whose type is assumed to be
-        ``torch.float32`` to half-precision floating point format (``torch.float16``).
-        It allreduces those ``float16`` gradient tensors. Once compressed gradient
-        tensors are allreduced, its then callback called ``decompress`` converts the
-        aggregated result back to ``float32`` and takes the mean.
-
-        Example::
-            >>> ddp_model._register_comm_hook(process_group, fp16_compress_hook)
+    This DDP communication hook implements a simple gradient compression
```

Comment:
These changes are fine because it's a small diff, but in general can we try to keep formatting changes in a separate diff? It makes the actual changes a lot easier to review. Thanks!

Reply:
Sure. I also prefer separating out these noises. The formatting was done unintentionally. Thanks for the reminder!